tests/integration/config/test.yaml

```
@@ -0,0 +1,93 @@
+        params: "$@system#model.parameters()"
+        # lr: "$((@system#batch_size * @trainer#devices)/256) * 0.1" # Compute LR dynamically for different batch sizes
+        # weight_decay: 0.0
+        # momentum: 0.9
+
+    scheduler:
+      _target_: torch.optim.lr_scheduler.StepLR
+      optimizer: "@system#optimizer"
+      step_size: 30
+
+    metrics:
+      train: 
```

yamllint

[error] trailing spaces

(trailing-spaces)